cli/tsc/00_typescript.js

```diff
@@ -21936,6 +21941,10 @@ function getSetExternalModuleIndicator(options) {
   switch (getEmitModuleDetectionKind(options)) {
     case 3 /* Force */:
       return (file) => {
+        if (isClassicScript(file)) {
+          file.externalModuleIndicator = void 0;
+          return;
+        }
```

Comment:
Actually upstream this. I don't know what branch we're working off https://github.com/denoland/TypeScript/branches